src/EA.Weee.Web/Areas/Scheme/Controllers/TransferEvidenceController.cs

```diff
@@ -219,7 +219,8 @@ public async Task<ActionResult> TransferredEvidence(Guid pcsId, Guid evidenceNot
                     noteData, TempData[ViewDataConstant.TransferEvidenceNoteDisplayNotification])
                 {
                     RedirectTab = redirectTab,
-                    SystemDateTime = currentDateTime
+                    SystemDateTime = currentDateTime, 
+                    IncludeHistory = true
```

Comment:
Can be removed a transfer note wont have the history its the original evidence note that has the history. George to update the wireframe.

Reply:
done